src/components/MultiSelectQuizQuestion.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+
+const MultiSelectQuizQuestion = ({
```

Comment:
unfortunately there's a bug in this component. the video won't upload currently, but I can't always select every box I want. Try clicking around to verify the behavior